server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMessagesMethodTest.java

```diff
@@ -23,6 +23,7 @@
 import static com.jayway.restassured.RestAssured.with;
```

Comment:
Moreover, how do the message saved as a draft looks like ?
Is it in the draft folder ? Is it read (flag) ? Recent ?
I think you need also a test on this...